.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "tabWidth": 2,
+  "singleQuote": false,
+  "trailingComma": "all",
+  "printWidth": 120,
+  "bracketSpacing": true
+}
```

Comment:
If we want to use something other then defaults, I recommend setting the config inside .eslintrc.js as per: https://github.com/prettier/eslint-plugin-prettier?tab=readme-ov-file#options
as I've found that the default of useprettierrc does not seem to work for me

(i.e. try changing the prettierrc settings and running pnpm lint...for me it does not complain when it should)

We may also have to import/use eslint-config-prettier to turn off conflicting formatting rules if needed.

Unless you want to take prettier out of eslint altogether in which case we'd need to adjust the package.json scripts to run prettier on lint explicitly